txnprovider/txpool/txpoolcfg/txpoolcfg.go

```diff
@@ -69,11 +70,11 @@ var DefaultConfig = Config{
 	LogEvery:               30 * time.Second,
 
 	PendingSubPoolLimit: 10_000,
-	BaseFeeSubPoolLimit: 10_000,
-	QueuedSubPoolLimit:  10_000,
+	BaseFeeSubPoolLimit: 30_000,
+	QueuedSubPoolLimit:  30_000,
```

Comment:
above defaults are changed to align with the values that were used prior to this PR coming from the `DefaultDeprecatedTxPoolConfig` set in this commit https://github.com/erigontech/erigon/commit/bca0d1d473eebb42a003d941d8bd9e1d9926d6a7